docs/src/introduction/scopes.md

```diff
@@ -0,0 +1,167 @@
+# Scopes
+
+Scopes might be the most important concept in Stack. We have put a ton of thought into how they work and how they should be used. Scopes are the backbone of Stack, and they are what make it just like any other programming language.
+
+## What is a Scope?
+
+Similar to other languages, a scope is a collection of variables. In Stack, a scope defines a variable as a key-value pair: symbol and expression. When a variable is defined, it is added to the current scope. When a variable is called, it is looked up in the current scope (by the [purification](/glossary.html#purification) step).
```

Comment:
> *'...a scope is a collection of variables. In Stack, a scope defines a variable as a key-value pair: symbol and expression.'*

This could be shorted into something like, *'a scope is a collection of symbol-expression pairs.'*